sdk/servicebus/azure-messaging-servicebus/TROUBLESHOOTING.md

```diff
@@ -0,0 +1,152 @@
+# Troubleshooting Service Bus issues
+
+This troubleshooting guide covers failure investigation techniques, common errors for the credential types in the Azure Service Bus Java client library, and mitigation steps to resolve these errors.
+
+## Table of contents
+- [Implicit prefetch issue in Sync ReceiverClient](#implicit-prefetch-issue-in-Sync-ReceiverClient)
+- [Troubleshoot ServiceBusProcessorClient issues](#troubleshoot-servicebus-issues)
+  - [Client hangs or stalls with a high prefetch and maxConcurrentCall value](#Client-hangs-or-stalls-with-a-high-prefetch-and-maxConcurrentCall-value)
+    - [Credit calculation issue](#credit-calculation-issue)
+- [Autocomplete issue](#autocomplete-issue)
+- [Migrate from legacy to new client library](#migrate-from-legacy-to-new-client-library)
+- [Enable and configure logging](#enable-and-configure-logging)
+    - [Configuring Log4J 2](#configuring-log4j-2)
+    - [Configuring logback](#configuring-logback)
+    - [Enable AMQP transport logging](#enable-amqp-transport-logging)
+    - [Reduce logging](#reduce-logging)
+- [Get additional help](#get-additional-help)
+    - [Filing GitHub issues](#filing-github-issues)
+
+## Implicit prefetch issue in Sync ReceiverClient
+Even after the application disables prefetch in the client builder, the `receiveMessages` API in 
+`ServiceBusReceiverClient` can re-enable prefetch implicitly, which may not be obvious.
+
+A detailed write-up on this issue is present in the [SyncReceiveAndPrefetch][SyncReceiveAndPrefetch] document present 
+in the docs folder.
+
+## Troubleshoot ServiceBus issues
+### Client hangs or stalls with a high prefetch and maxConcurrentCall value
+Client throws `Update disposition request timed out.` and hangs or stalls processing new messages. 
+This issue is known to occur when the number of threads in threadpool is low or equal to the `maxConcurentCalls` value.
+
+Mitigation : add vm option `-Dreactor.schedulers.defaultBoundedElasticSize=<large number greater than concurrency count>`
+The default value of this property is `10 * number of CPU cores`. This problem is encountered more 
+frequently in an AKS environment.
```

Comment:
I'd list out what AKS is first. `... Azure Kubernetes Service (AKS) environment`